tsl/src/bgw_policy/job.c

```diff
@@ -250,9 +273,15 @@ bool
 policy_retention_execute(int32 job_id, Jsonb *config)
 {
 	PolicyRetentionData policy_data;
+	bool verbose_log;
+	int elevel;
 
 	policy_retention_read_and_validate_config(config, &policy_data);
 
+	verbose_log = policy_get_verbose_log(config);
+	elevel = (verbose_log) ? LOG : DEBUG1;
```

Comment:
I think the original idea was not to have `DEBUG` vs `LOG` but simply the following kind of thing:
```
		if (verbose_log)
			elog(LOG,
```

Reply:
I did it this way because it's similar to what's done in `continuous_agg_refresh_internal`:
```
log_refresh_window(callctx == CAGG_REFRESH_POLICY ? LOG : DEBUG1, 
cagg, 
&refresh_window, 
"refreshing continuous aggregate");
```
Do you think the DEBUG should be completely removed?

Reply:
I think we don't need to complicate things here and do the same what we already does with `log_refresh_window`

Reply:
@gayyappan and @svenklemm 's proposal was to not do that though, and only use `LOG` instead, and this is why I suggested it. You can look how it was done in https://github.com/timescale/timescaledb/pull/3520/files

Reply:
Ahh now I got what u meant... I was looking into our `master` branch and didn't found this `verbose_log` usage because after the PR you mentioned we changed the `policy_compression_execute` re-implementing it as a PL/pgSQL procedure instead of C code and we loose it in our code base.

Anyway the usage on the procedure is similar what you said: https://github.com/timescale/timescaledb/blob/master/sql/policy_internal.sql#L79